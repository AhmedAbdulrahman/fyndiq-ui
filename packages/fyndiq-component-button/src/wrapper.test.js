import React from 'react'
import { shallow } from 'enzyme'

import Wrapper from './wrapper'

describe('fyndiq-component-button Wrapper', () => {
  it('should have the proper structure', () => {
    expect(shallow(<Wrapper />)).toMatchSnapshot()
  })

  it('should have an orientation prop', () => {
    expect(
      shallow(<Wrapper orientation="vertical" />).hasClass(
        'wrapper--orientation_vertical',
      ),
    ).toBe(true)
  })

  it('should have a className prop', () => {
    expect(
      shallow(<Wrapper className="className" />).hasClass('className'),
    ).toBe(true)
  })

  it('should have a noOuterBorder prop', () => {
    expect(
      shallow(<Wrapper noOuterBorder />).hasClass('wrapperNoOuterBorder'),
    ).toBe(true)
  })
})
