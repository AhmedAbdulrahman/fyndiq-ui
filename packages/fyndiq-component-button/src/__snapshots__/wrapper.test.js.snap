// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fyndiq-component-button Wrapper should have the proper structure 1`] = `
<div
  className="
      wrapper
      wrapper--orientation_horizontal
      false
      
    "
/>
`;
